### ISSUE TITLE
Support filtering memos by prefix

### ISSUE THREAD

**Issue by wenlzhang** [Quorafind/Obsidian-Thino#134]
Suppose we have the following types of memos:
```
- 12:00 List item A
- [ ] 12:01 Uncompleted task B
- [x] 12:02 Completed task C
- [>] 12:03 Re-arranged task D
```
Then:

- Filtering for the prefix `-` or `- ` would give all the four memos.
- Filtering for the prefix `- [` would give all tasks, regardless of their status.
- Filtering for the prefix `- [ ]` would give memo B.
- Filtering for the prefix `- [x]` would give memo C.

This prefix filtering can be a separate type, in addition to the existing filter types. If this can be supported, Memos can be used to conveniently manage different types of items/tasks.

The following issue is related to this:
https://github.com/Quorafind/Obsidian-Memos/issues/32

**Comment by Quorafind** [comment]
Available in next version

**Comment by Quorafind** [comment]
Sorry, I need to released this in 1.9.1 or 1.9.2

**Comment by Quorafind** [comment]
I'm very sorry for not replying to you for so long. The latest version of Thino (Obsidian Memos) now supports this feature.
